Frontend/ui-driver/src/Flow.purs

```diff
@@ -872,13 +884,17 @@ applicationSubmittedFlow screenType = do
 driverProfileFlow :: FlowBT String Unit
 driverProfileFlow = do
   logField_ <- lift $ lift $ getLogFields
+  config <- getAppConfigFlowBT Constants.appConfig
```

Comment:
if the value of config is only required in case of `GO_TO_REFERRAL_SCREEN_FROM_DRIVER_PROFILE_SCREEN` can you move it to the respective scope?